examples/config-sync.json

```diff
@@ -13,54 +13,7 @@
 	"extensions": {
 		"sync": {
 			"enable": true,
-			"credentialsFile": "./examples/sync-auth-filepath.json",
 			"registries": [
-				{
```

Comment:
Why is this removed? Has anything changed with the format?

Reply:
by mistake, will revert.